WcaOnRails/app/assets/stylesheets/collapsible-panels.scss

```diff
@@ -15,7 +15,8 @@
  * Make collapse link fill up entire panel-title
  * Copied from http://stackoverflow.com/a/20188613
  */
-.panel-title a[data-toggle="collapse"] {
+.panel-title a[data-toggle="collapse"],
+.panel-title a.full-space {
```

Comment:
Why that selector change?
Isn't there any other place where we would need to add this "full-space" class?

Reply:
This makes an `a` tag take the whole space of a panel title.
The first selector applies to `a` tags that `collapse` the given panel back and forth with some bootstrap JavaScript. I wanted to reuse the same styling for the Bylaws panel, but it doesn't have `[data-toggle="collapse"]`, it's just a normal link!